cmake/ConfigCPack.cmake

```
@@ -171,6 +171,11 @@ if(ROCPROFSYS_USE_MPI)
         list(APPEND _DEBIAN_PACKAGE_DEPENDS "libmpich-dev")
     endif()
 endif()
+if(ROCPROFSYS_BUILD_TESTING)
+    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocdecode")
+    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocdecode-dev")
+    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocdecode-test")
+endif()
```

```suggestion
if(ROCPROFSYS_BUILD_TESTING)
    list(APPEND _DEBIAN_PACKAGE_DEPENDS "rocdecode-test")
endif()
```
You should be able to just include "rocdecode-test", this package will take care of the other dependencies.